mordant/src/jvmMain/kotlin/com/github/ajalt/mordant/internal/jna/JnaMacosMppImpls.kt

```diff
@@ -1,9 +1,12 @@
-package com.github.ajalt.mordant.internal
+package com.github.ajalt.mordant.internal.jna
 
+import com.github.ajalt.mordant.internal.MppImpls
+import com.oracle.svm.core.annotate.Delete
 import com.sun.jna.*
 import java.io.IOException
 import java.util.concurrent.TimeUnit
 
+@Delete
```

Comment:
I see you made `PosixLibC` private. Should this be as well?

Reply:
Good catch :+1: